banzai/images.py

```diff
@@ -96,24 +98,25 @@ def writeto(self, filename, fpack=False):
             bpm_hdu = fits.ImageHDU(self.bpm.astype(np.uint8))
             bpm_hdu.name = 'BPM'
             hdu_list.append(bpm_hdu)
+        hdu_list = self.add_astropy_data_tables_to_hdu_list_to_be_saved(hdu_list)
```

Comment:
For consistency, you could also refactor the catalog and bpm additions to their own methods

Reply:
will do

Reply:
I would prefer to refactor to just data table. In principle this could be a pandas dataframe, etc. Nothing we do here is astropy specific (I think).